check

```diff
@@ -265,7 +287,7 @@ def do_checks(srcdir):
         if 'domains' in data:
             errcnt += check_rdns(data)
             for domain in data.get('domains', []):
-                errcnt += check_dupe("Domain", domain, domains, community)
+                errcnt += check_dupe("Domain", domain.upper(), domains, community)
```

Comment:
```
try:
    domain.encode("ascii")
except UnicodeEncodeError:
    errcnt += 1
    error("Domain not IDNA-encoded: %s" % (domain))
```
To prevent umlauts in domains from community-files.